exporters/trace/README.md

```diff
@@ -1 +1,47 @@
 # Cloud Trace Exporter for OpenTelemetry
+
+  OpenTelemetry Google Cloud Trace Exporter allows the user to send collected traces to Google Cloud. 
+  
+  <a href="https://cloud.google.com/trace">Google Cloud Trace</a> is a distributed tracing backend system. It helps developers to gather timing data needed to troubleshoot latency problems in microservice & monolithic architectures. It manages both the collection and lookup of gathered trace data.
+
+## Setup
+  Google Cloud Trace is a managed service provided by Google Cloud Platform.
+
+## Installation
+  Not currently ready as a package to be imported. To use presently, one would have to clone this GitHub repo.
+## Usage
+  If you are running in a GCP environment, the exporter will automatically authenticate using the environment's service account. If not, you will need to follow the instructions in Authentication.  
+    
+  The TraceExporter constructor takes in three parameters: String for projectId, TraceServiceClient to create a traceServiceClient instance, and Map<String, AttributeValue> for the fixed attributes of a span.  
+  So, we need to import the following: 
+  ```java
+  import com.google.cloud.trace.v2.TraceServiceClient;
+  import com.google.devtools.cloudtrace.v2.AttributeValue;
+  import com.google.devtools.cloudtrace.v2.ProjectName;
+  import com.google.devtools.cloudtrace.v2.Span;
+  ```
+  Declare and initialize the variables that will be used in the constructor parameters.  
+  Then, we can create a TraceExporter with, for example:
+  ```java
+  TraceExporter exporter = new TraceExporter(projectId, traceServiceClient, fixedAttributes);
+  ```
+  Start tracing and collecting SpanData. To export these spans, use the export method, where spanDataList is a Collection of SpanData already collected:
+  ```java
+  exporter.export(spanDataList)
```

Comment:
We don't expect end users to export spans directly via our exporter.
Instead, they should configure the trace provider with the exporter, see example: https://github.com/open-telemetry/opentelemetry-java/blob/a59748a904e0cb1e2e4f8df50dd346a82f26ff1e/examples/jaeger/src/main/java/io/opentelemetry/example/JaegerExample.java#L39-L41

The rest of the code shouldn't be specific to Cloud Trace.
They should just create spans using global `opentelemetry-java` API packages, e.g. https://github.com/open-telemetry/opentelemetry-java/blob/a59748a904e0cb1e2e4f8df50dd346a82f26ff1e/examples/jaeger/src/main/java/io/opentelemetry/example/JaegerExample.java#L46

Reply:
Ahh okay that makes sense; thanks for the example links too. Changed.